client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -398,7 +400,7 @@ export class List extends React.Component {
 				<div className="list__header-primary-domain-content">
 					<ExternalLink
 						className="list__header-primary-domain-url"
-						href={ selectedSite.URL }
+						href={ logmeinUrl( selectedSite.URL, siteUrls ) }
```

Comment:
Would you like to include in this PR the logmeinUrl()'ification of other instances in Calypso that link to the mapped domain URL? We might have to do this in the editor, reader and so on. Or do you think we should take that as a separate PR?